tools/sigclient/pkg/ingest/ingest.go

```diff
@@ -356,7 +357,10 @@ func getReaderFromArgs(iType IngestType, nummetrics int, gentype string, str str
 		rdr = utils.InitDynamicUserGenerator(ts, seed, generatorDataConfig)
 	case "benchmark-ingest-query":
 		log.Infof("Initializing benchmark-ingest-query reader")
-		seed := int64(1001)
+		source := rand.NewSource(time.Now().UnixNano())
+		randomNumGen := rand.New(source)
+		lowerBound := 1001
+		seed := int64(lowerBound + randomNumGen.Intn(10000)) // Generate random number between 1000 and 11000
```

Comment:
What's the purpose of this lower bound?

Reply:
I thought all other gentypes are using 1000 or 1001 as seed so I added that as lowerbound but it does not matter so I removed it now.